src/kili/services/asset_import/legacy.py

```diff
@@ -150,7 +150,7 @@ def process_json_content(
         if is_url(json_content):
             processed_json_content_array.append(json_content)
         else:
-            if input_type in ("FRAME", "VIDEO"):
+            if input_type in ("VIDEO_LEGACY", "VIDEO"):
```

Comment:
this should be only VIDEO.
But I think that we can delete the legacy importer now. We will recode the one from time series when prioritized later ?